app/notify_client/status_api_client.py

```diff
@@ -10,5 +10,9 @@ def get_status(self, *params):
     def get_count_of_live_services_and_organisations(self):
         return self.get(url='/_status/live-service-and-organisation-counts')
 
+    @cache.set('live-service-and-organisation-counts', ttl_in_seconds=3600)
+    def get_count_of_live_services_and_organisations_cached(self):
+        return self.get(url='/_status/live-service-and-organisation-counts')
+
 
```

Comment:
this is for the redis healthcheck, if manually setting a key succeeds, we try the cache decorator